src/AppBundle/Controller/WebhookController.php

```diff
@@ -18,8 +18,18 @@ public function githubAction(Request $request)
     {
         $data = json_decode($request->getContent(), true);
         if ($data === null) {
-            throw new \Exception('Invalid JSON body!');
+            return new JsonResponse('Invalid JSON body!', 400);
         }
+        if (!isset($data['repository']['full_name'])) {
+            return new JsonResponse('Repository name not defined!', 400);
+        }
+
+        list($owner, $name) = explode('/', $data['repository']['full_name']);
```

Comment:
Github uses ``owner`` (see [here](https://github.com/carsonbot/carsonbot/blob/master/src/AppBundle/Tests/webhook_examples/pull_request.opened.json#L25)) but I agree that ``vendor`` seems to be better.